src/contexts/SwapContext.tsx

```diff
@@ -126,6 +135,7 @@ export const initialSwapContext: ISwapContext = {
     priorityFeeInSOL: 0,
     setPriorityFeeInSOL() {},
   },
+  setUserSlippage: () => {},
```

Comment:
prefer to put this to `null` | `ISwapContext`

Reply:
can you clarify what this means?